masq_lib/src/percentage.rs

```diff
@@ -421,78 +512,66 @@ mod tests {
     #[test]
     fn should_be_rounded_to_works_for_last_but_one_digit() {
         [
-            (49, RoundingTo::SmallerPositive, RoundingTo::SmallerNegative),
-            (50, RoundingTo::BiggerPositive, RoundingTo::BiggerNegative),
-            (51, RoundingTo::BiggerPositive, RoundingTo::BiggerNegative),
-            (5, RoundingTo::SmallerPositive, RoundingTo::SmallerNegative),
-            (
-                100,
-                RoundingTo::SmallerPositive,
-                RoundingTo::SmallerNegative,
-            ),
-            (
-                787879,
-                RoundingTo::BiggerPositive,
-                RoundingTo::BiggerNegative,
-            ),
-            (
-                898784545,
-                RoundingTo::SmallerPositive,
-                RoundingTo::SmallerNegative,
-            ),
+            (49, 0),
+            (50, 1),
+            (51, 1),
+            (5, 0),
+            (99,1),
+            (0,0)
```

Comment:
This is simple, yet so effective in explaining what's going on... nicely done @bertllll.